components/button-cards/generic-card/generic-card-media-floating-icon.hbs

```diff
@@ -0,0 +1,45 @@
+<h2>Large Media cards</h2>
+<div class="row card-row">
+    <div class="col-12 col-md-4">
+        <div class="msds-generic-card msds-generic-card--media">
+            <a href="#" class="msds-generic-card--link">
+
+                <div class="msds-generic-card__frame embed-responsive embed-responsive-2by1">
+                    <div class="msds-generic-card__media-container">
+                        <div class="msds-generic-card__floating-icon-container">
+                            <svg class="msds-generic-card__floating-icon">
+                                <use class="msds-generic-card__floating-icon"
```

Comment:
Does it need to have the same class as its SVG parent element? or can it be removed?